runtime/tutor.txt

```diff
@@ -416,6 +415,338 @@ _________________________________________________________________
+
+
+
+
+
+
+
+=================================================================
+=                   COPYING AND PASTING TEXT                    =
+=================================================================
+
+ Press y to yank (copy) the selection.
+ Press p to paste the yanked selection after the cursor.
+ Press uppercase P to paste the yanked selection *before*
+ the cursor.
+
+ 1. Move the cursor to the line below marked -->.
+    Make sure your cursor is on the "b" of banana.
+ 2. Press w to select "banana" and y to yank it.
+ 3. Move to the space between "2" and "3" and press p to paste.
+ 4. Repeat between "3" and "4".
+
+ --> 1 banana 2 3 4
+
+ Whenever you delete (d) or change (c) text, Helix will copy the
+ altered text. You can avoid this with alt-d/c respectively.
+ Also, Helix doesn't share the system clipboard by default. Press
+ space-y and space-p to yank/paste to/from your computers main
+ clipboard.
+
+=================================================================
+=                        SWITCHING CASE                         =
+=================================================================
+
+ Press ~ to switch the case of all selected letters.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Select each wrongly capitalised or lowercase letter
+    and press ~ over them.
+ (remember, the first letter *should* be capitalised)
+
+ --> thIs sENtencE hAs MIS-cApitalIsed leTTerS.
+
+
+
+
+
+
+
+
+
+
+=================================================================
+=                         SETTING CASE                          =
+=================================================================
+
+ Press ` (above tab key on most keyboards) to set all selected
+ letters to lowercase.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Press x to select the line.
+ 3. Press ` to change it to all lowercase.
+
+--> this SENTENCE SHOULD all be in LOWERCASE!
+
+ Note: Conversely, alt-` changes all selected letters to uppercase.
+
+
+
+
+
+
+
+
+=================================================================
+=                            MACROS                             =
+=================================================================
+
+ Macros are a way to record a set of actions that you want
+ to repeat.
+
+ Press Q to start recording a macro, you should see a popup at
+ the bottom of your screen. Press Q again to stop recording.
+ Press q to repeat your recorded macro.
+
+ 1. Move the cursor to the first line below marked -->.
+    Ensure your cursor is on the > of the arrow.
+ 2. Press Q to start recording.
+ 3. Edit the line to look like the bottom one.
+ 4. Exit insert and press Q again to stop recording.
+ 5. Move to the line below and put your cursor on the > again.
+ 6. Press q to repeat the macro.
+
+ --> ... sentence doesn't have it's first and last ... .
+ --> ... sentence doesn't have it's first and last ... .
+ --> This sentence doesn't have it's first and last word.
```

Comment:
We can have a tip here to on recording macros to a specific register.

Reply:
Yeah, the problem is that this section is already almost too large to fit on the screen (see Omnikar's first comment) as is the copying/pasting section where I also avoid mentioning registers. Register's could be made into it's own section and possibly cover both copying and macros?

Reply:
I like that idea for introducing registers in their own section.